src/components/Products/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from "react";
+import { useSelector } from "react-redux";
+import { Row, Col } from 'react-bootstrap';
+import Product from "./Product";
+import useProductHooks from "../../redux/Products/products-actions";
+import { ProductStyled, ProductTitle, ProductHeading, ProductParagraph } from './product.style'
+
+const Products = () => {
+
+  const allProducts = useSelector(state => state.getProducts);
+  const { products, loading, error } = allProducts;
+  const { getProducts } = useProductHooks();
+
+  useEffect(() => {
+    getProducts();
+  }, []);
```

Comment:
Add check to load products from redux, if not then send db call
Also look for updates